src/test/java/com/znsio/teswiz/steps/VodQASteps.java

```diff
@@ -0,0 +1,38 @@
+package com.znsio.teswiz.steps;
+
+import com.context.SessionContext;
+import com.context.TestExecutionContext;
+import com.znsio.teswiz.businessLayer.vodqa.VodqaBL;
+import com.znsio.teswiz.entities.SAMPLE_TEST_CONTEXT;
+import com.znsio.teswiz.runner.Drivers;
+import com.znsio.teswiz.runner.Runner;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.log4j.Logger;
+
+public class VodQASteps {
+    private static final Logger LOGGER = Logger.getLogger(TheAppSteps.class.getName());
+    private final TestExecutionContext context;
+
+    public VodQASteps() {
+        context = SessionContext.getTestExecutionContext(Thread.currentThread().getId());
+        LOGGER.info("context: " + context.getTestName());
+    }
+
+    @Given("I login into the application")
+    public void iLoginIntoTheApplication() {
+        Drivers.createDriverFor(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform(), context);
+        new VodqaBL().login();
+    }
+
+    @When("I tap in the middle of the screen")
+    public void iTapInTheMiddleOfTheScreen() {
+        new VodqaBL().tapInTheMiddleOfTheScreen();
+    }
+
+    @Then("I am able to view {string} page")
+    public void iAmAbleToViewElement(String pageHeader) {
+        new VodqaBL().verifyPageHeader(pageHeader);
```

Comment:
pass context and platform to BL constructor as arguments
new VodqaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).verifyPageHeader(pageHeader);